[2022-06-29 17:17:29.245] [remoteagent] [info] Extension host agent started.
[2022-06-29 17:17:29.977] [remoteagent] [info] [127.0.0.1][6ec3e844][ManagementConnection] New connection established.
[2022-06-29 17:17:30.456] [remoteagent] [info] [127.0.0.1][bcc73b2d][ExtensionHostConnection] New connection established.
[2022-06-29 17:17:30.465] [remoteagent] [info] [127.0.0.1][bcc73b2d][ExtensionHostConnection] <1300> Launched Extension Host Process.
[2022-06-29 17:18:08.384] [remoteagent] [info] [127.0.0.1][83dcb3c2][ManagementConnection] New connection established.
[2022-06-29 17:18:09.090] [remoteagent] [info] [127.0.0.1][ffce907b][ExtensionHostConnection] New connection established.
[2022-06-29 17:18:09.100] [remoteagent] [info] [127.0.0.1][ffce907b][ExtensionHostConnection] <1460> Launched Extension Host Process.
[2022-06-29 17:18:49.646] [remoteagent] [info] [127.0.0.1][ffce907b][ExtensionHostConnection] The client has reconnected.
[2022-06-29 17:31:42.491] [remoteagent] [error] ["Error: Could not find pty on pty host","    at w._throwIfNoPty (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:5333)","    at w.updateIcon (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:1733)","    at Object.call (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:13:8386)","    at h.onPromise (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4891)","    at h.onRawMessage (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4274)","    at /home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:3560","    at E.invoke (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:145)","    at W.deliver (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:2271)","    at c.fire (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:1849)","    at process.U (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:6:35698)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-06-29 17:31:42.493] [remoteagent] [error] ["Error: Could not find pty on pty host","    at w._throwIfNoPty (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:5333)","    at w.updateTitle (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:1673)","    at Object.call (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:13:8386)","    at h.onPromise (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4891)","    at h.onRawMessage (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4274)","    at /home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:3560","    at E.invoke (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:145)","    at W.deliver (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:2271)","    at c.fire (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:1849)","    at process.U (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:6:35698)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-06-29 17:31:42.494] [remoteagent] [error] ["Error: Could not find pty on pty host","    at w._throwIfNoPty (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:5333)","    at w.updateTitle (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:1673)","    at Object.call (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:13:8386)","    at h.onPromise (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4891)","    at h.onRawMessage (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4274)","    at /home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:3560","    at E.invoke (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:145)","    at W.deliver (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:2271)","    at c.fire (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:1849)","    at process.U (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:6:35698)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-06-29 17:33:38.270] [remoteagent] [info] [127.0.0.1][83dcb3c2][ManagementConnection] The client has reconnected.
[2022-06-29 17:33:38.747] [remoteagent] [info] [127.0.0.1][ffce907b][ExtensionHostConnection] The client has reconnected.
[2022-06-29 17:33:39.783] [remoteagent] [error] ["Error: Could not find pty on pty host","    at w._throwIfNoPty (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:5333)","    at w.updateTitle (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:1673)","    at Object.call (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:13:8386)","    at h.onPromise (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4891)","    at h.onRawMessage (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4274)","    at /home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:3560","    at E.invoke (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:145)","    at W.deliver (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:2271)","    at c.fire (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:1849)","    at process.U (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:6:35698)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-06-29 17:35:41.453] [remoteagent] [info] [127.0.0.1][83dcb3c2][ManagementConnection] The client has reconnected.
[2022-06-29 17:35:42.728] [remoteagent] [error] ["Error: Could not find pty on pty host","    at w._throwIfNoPty (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:5333)","    at w.updateIcon (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:1733)","    at Object.call (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:13:8386)","    at h.onPromise (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4891)","    at h.onRawMessage (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4274)","    at /home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:3560","    at E.invoke (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:145)","    at W.deliver (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:2271)","    at c.fire (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:1849)","    at process.U (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:6:35698)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-06-29 17:35:42.729] [remoteagent] [error] ["Error: Could not find pty on pty host","    at w._throwIfNoPty (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:5333)","    at w.updateTitle (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:1673)","    at Object.call (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:13:8386)","    at h.onPromise (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4891)","    at h.onRawMessage (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4274)","    at /home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:3560","    at E.invoke (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:145)","    at W.deliver (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:2271)","    at c.fire (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:1849)","    at process.U (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:6:35698)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-06-29 17:35:42.729] [remoteagent] [error] ["Error: Could not find pty on pty host","    at w._throwIfNoPty (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:5333)","    at w.updateTitle (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:1673)","    at Object.call (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:13:8386)","    at h.onPromise (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4891)","    at h.onRawMessage (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4274)","    at /home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:3560","    at E.invoke (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:145)","    at W.deliver (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:2271)","    at c.fire (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:1849)","    at process.U (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:6:35698)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-06-29 17:35:42.936] [remoteagent] [error] ["Error: Could not find pty on pty host","    at w._throwIfNoPty (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:5333)","    at w.updateTitle (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:1673)","    at Object.call (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:13:8386)","    at h.onPromise (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4891)","    at h.onRawMessage (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4274)","    at /home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:3560","    at E.invoke (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:145)","    at W.deliver (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:2271)","    at c.fire (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:1849)","    at process.U (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:6:35698)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-06-29 17:35:50.729] [remoteagent] [info] [127.0.0.1][ffce907b][ExtensionHostConnection] The client has reconnected.
[2022-06-29 18:06:37.601] [remoteagent] [info] [127.0.0.1][f8889cf7][ManagementConnection] New connection established.
[2022-06-29 18:06:38.088] [remoteagent] [info] [127.0.0.1][15072839][ExtensionHostConnection] New connection established.
[2022-06-29 18:06:38.103] [remoteagent] [info] [127.0.0.1][15072839][ExtensionHostConnection] <4495> Launched Extension Host Process.
[2022-06-29 18:06:38.635] [remoteagent] [info] [127.0.0.1][6ec3e844][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
[2022-06-29 18:06:38.638] [remoteagent] [error] Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/server/node/server.main.js:143:7923)
    at process.emit (node:events:390:28)
[2022-06-29 18:08:00.255] [remoteagent] [info] [127.0.0.1][6ec3e844][ManagementConnection] Another client has connected, will shorten the wait for reconnection 5m before disposing...
[2022-06-29 18:08:00.257] [remoteagent] [info] [127.0.0.1][01eb8aa9][ManagementConnection] New connection established.
[2022-06-29 18:08:00.689] [remoteagent] [info] [127.0.0.1][476012c2][ExtensionHostConnection] New connection established.
[2022-06-29 18:08:00.696] [remoteagent] [info] [127.0.0.1][476012c2][ExtensionHostConnection] <4686> Launched Extension Host Process.
[2022-06-29 18:08:06.500] [remoteagent] [error] ["Error: Could not find pty on pty host","    at w._throwIfNoPty (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:5333)","    at w.updateIcon (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:1733)","    at Object.call (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:13:8386)","    at h.onPromise (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4891)","    at h.onRawMessage (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4274)","    at /home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:3560","    at E.invoke (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:145)","    at W.deliver (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:2271)","    at c.fire (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:1849)","    at process.U (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:6:35698)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-06-29 18:11:37.893] [remoteagent] [info] [127.0.0.1][bcc73b2d][ExtensionHostConnection] <1300> Extension Host Process exited with code: 0, signal: null.
[2022-06-29 18:13:00.390] [remoteagent] [info] [127.0.0.1][6ec3e844][ManagementConnection] The reconnection short grace time of 5m has expired, so the connection will be disposed.
[2022-06-29 18:13:11.501] [remoteagent] [info] [127.0.0.1][83dcb3c2][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
[2022-06-29 18:13:11.501] [remoteagent] [error] Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/ubuntu/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/server/node/server.main.js:143:7923)
    at process.emit (node:events:390:28)
[2022-06-29 18:21:02.433] [remoteagent] [info] [127.0.0.1][f8889cf7][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
[2022-06-29 18:21:02.547] [remoteagent] [info] [127.0.0.1][15072839][ExtensionHostConnection] <4495> Extension Host Process exited with code: 0, signal: null.
[2022-06-29 18:21:27.665] [remoteagent] [info] [127.0.0.1][01eb8aa9][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
[2022-06-29 18:21:27.762] [remoteagent] [info] [127.0.0.1][476012c2][ExtensionHostConnection] <4686> Extension Host Process exited with code: 0, signal: null.
[2022-06-29 18:52:23.908] [remoteagent] [info] [127.0.0.1][83dcb3c2][ManagementConnection] Another client has connected, will shorten the wait for reconnection 5m before disposing...
[2022-06-29 18:52:23.909] [remoteagent] [info] [127.0.0.1][45b49370][ManagementConnection] New connection established.
[2022-06-29 18:52:24.608] [remoteagent] [info] [127.0.0.1][4fa3eb81][ExtensionHostConnection] New connection established.
[2022-06-29 18:52:24.621] [remoteagent] [info] [127.0.0.1][4fa3eb81][ExtensionHostConnection] <5082> Launched Extension Host Process.
[2022-06-29 18:57:24.038] [remoteagent] [info] [127.0.0.1][83dcb3c2][ManagementConnection] The reconnection short grace time of 5m has expired, so the connection will be disposed.
[2022-06-29 18:57:24.068] [remoteagent] [info] [127.0.0.1][ffce907b][ExtensionHostConnection] <1460> Extension Host Process exited with code: 0, signal: null.
[2022-06-29 19:15:11.537] [remoteagent] [info] [127.0.0.1][15cddcae][ManagementConnection] New connection established.
[2022-06-29 19:15:11.981] [remoteagent] [info] [127.0.0.1][6e101aa6][ExtensionHostConnection] New connection established.
[2022-06-29 19:15:11.990] [remoteagent] [info] [127.0.0.1][6e101aa6][ExtensionHostConnection] <5769> Launched Extension Host Process.
[2022-06-29 19:38:58.822] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-06-29 19:46:31.607] [remoteagent] [error] No ptyHost heartbeat after 12 seconds
